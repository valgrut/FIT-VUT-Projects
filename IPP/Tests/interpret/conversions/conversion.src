.IPPcode18 # koment
DEFVAR GF@cislo
DEFVAR GF@znak
DEFVAR GF@result

# INT2CHAR
WRITE string@Napiš\032číslo:
READ GF@cislo int
INT2CHAR GF@result GF@cislo
write string@výsledný\032znak\032je:
WRITE GF@result

# STRI2INT
WRITE string@Napiš\032znak:
READ GF@znak string
STRI2INT GF@result GF@znak int@0
write string@výsledné\032číslo\032je: 
WRITE GF@result

