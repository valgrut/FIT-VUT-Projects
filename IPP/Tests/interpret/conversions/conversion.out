0
Napiš číslo:
výsledný znak je:
B
Napiš znak:
výsledné číslo je:
97
